### PR TITLE
Pmp2

### PR DESCRIPTION
**Pull request by popkinj**
# Fixes to Migrations
- The spatial migrations are now working.
- Duplicate column entries commented out

### CODE REVIEW

File: database/src/migrations/20210622120101_activity_incoming_data.ts [hunk 1]
```diff
@@ -24,18 +24,18 @@ export async function up(knex: Knex): Promise<void> {
     
     COMMENT ON COLUMN ${DB_SCHEMA}.activity_incoming_data.sync_status IS 'Sync/Save status of the activity form';
 
-    ALTER TABLE ${DB_SCHEMA}.activity_incoming_data
-    ADD COLUMN review_status VARCHAR(100) DEFAULT 'Not Reviewed';
+    -- ALTER TABLE ${DB_SCHEMA}.activity_incoming_data
```
Comment (micheal-w-wells) [inline database/src/migrations/20210622120101_activity_incoming_data.ts]: is this intentional?

Comment (popkinj) [reply database/src/migrations/20210622120101_activity_incoming_data.ts]: Good eye... Yes. They causing failures. Already present it said.

Comment (popkinj) [reply database/src/migrations/20210622120101_activity_incoming_data.ts]: I'll double check that.

Comment (popkinj) [reply database/src/migrations/20210622120101_activity_incoming_data.ts]: Don't accept the PR yet... This is weird.
Firing up the brand new docker image and all those columns were there. The ones I commented out that is.

Comment (popkinj) [reply database/src/migrations/20210622120101_activity_incoming_data.ts]: Do the migrations run automatically?

Comment (popkinj) [reply database/src/migrations/20210622120101_activity_incoming_data.ts]: Hmmm... I'm wondering if that table quit halfway through. Which I didn't think Knex allowed.

Comment (popkinj) [reply database/src/migrations/20210622120101_activity_incoming_data.ts]: K... Don't know what happened. I searched the directory and there's no duplication. So those statements are legit. Just pushed up the correction.

Comment (micheal-w-wells) [reply database/src/migrations/20210622120101_activity_incoming_data.ts]: they should run automaticaly on pr merge yes

**Review by micheal-w-wells — APPROVED** [review]

**Pull request merged**